mw-webapp/src/component/input/Input.module.scss

```diff
@@ -0,0 +1,30 @@
+@import "src/styles/variables.scss";
+
+.input_wrapper {
+  position: relative;
+  font-size: 1.4rem;
+  font-style: normal;
+  font-weight: 400;
+}
+
+.visually-hidden {
+  position: fixed;
+  transform: scale(0);
```

Comment:
could you explain this trick, why do we need it?

Reply:
https://www.w3.org/WAI/tutorials/forms/labels/
![image](https://github.com/tritonJS826/masters-way/assets/32575876/f2b238ae-063e-43aa-80a7-743b08a11c23)

Reply:
don't understand why do you need to hide something, sory not clear from this screenshot

Reply:
This is necessary for accessibility.  If a person uses a screen reader, he will understand what it is, I can remove it, leave one return input and not worry about it for now. The label will be on the page, but it will be hidden for the user and readable for the screen reader, since it is in the dom